webapi/src/main/scala/org/knora/webapi/slice/api/v3/ontology/OntologyRestServiceV3.scala

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright © 2021 - 2025 Swiss National Data and Service Center for the Humanities and/or DaSCH Service Platform contributors.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.knora.webapi.slice.api.v3.ontology
+import zio.*
+
+import org.knora.webapi.messages.IriConversions.ConvertibleIri
+import org.knora.webapi.messages.OntologyConstants.Rdfs
+import org.knora.webapi.messages.SmartIri
+import org.knora.webapi.messages.StringFormatter
+import org.knora.webapi.messages.v2.responder.ontologymessages.ReadClassInfoV2
+import org.knora.webapi.slice.api.v3.LanguageStringDto
+import org.knora.webapi.slice.api.v3.NotFound
+import org.knora.webapi.slice.api.v3.OntologyDto
+import org.knora.webapi.slice.api.v3.ResourceClassDto
+import org.knora.webapi.slice.common.KnoraIris.OntologyIri
+import org.knora.webapi.slice.common.KnoraIris.ResourceClassIri
+import org.knora.webapi.slice.ontology.domain.service.OntologyRepo
+
+class OntologyRestServiceV3(ontologyRepo: OntologyRepo)(implicit val sf: StringFormatter) {
```

Comment:
My understanding so far has been, that the RestService would call a Service, not a Repo

Reply:
Maybe we should rename the current `OntologyRepo` to `OntologyService` then.